### PR TITLE
Txt2markdown

### PR DESCRIPTION
**Pull request by MelonSmasher**
* Created a readme.md file. 
* I've fixed all indents for each section and it's children. 
* I've converted ordered lists that are not recognized by markdown to non-ordered lists that are recognized by markdown. See Figure 1.
* Ordered lists that are on the 3 tier in the hierarchy have been kept as ordered lists. See Figure 1.

#### Before:
```text
B. Division Changes
        1. The bottom team in each division will rotate to another division.
            a) North → East → South → West

```

#### After:
````markdown
### B. Division Changes

  1. The bottom team in each division will rotate to another division.
    * North → East → South → West
```

### CODE REVIEW

**Comment by MelonSmasher** [comment]
@beaubreedlove take a look before I merge.

**Comment by beaubreedlove** [comment]
Looks good. Some of the empty headings look weird because they are empty. I'm not sure how I feel about it yet. I sort of want to do some reorganization before we commit this change.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+  * If a player is assigned two positions, his team’s manager may determine which position he plays at any given time.
+3. Each of the players in play on a team must fit into one of following slots, and no slot may hold more than one player:
+  * 1 Quarterback (QB)
+  * 2 Running Backs (RB)
+  * 2 Wide Receivers (WR)
+  * 1 Tight End (TE)
+  * 1 Kicker (K)
+  * 1 Defense + Special Teams (DST/ST)
+  * 1 Flex (FLEX)
+    * A running back, wide receiver, or tight end may fill this slot.'
```
Comment (beaubreedlove) [inline README.md]: extra `'`

**Comment by MelonSmasher** [comment]
I was thinking that those headings needed to be filled out. I wasn't sure if you wanted it to be in a separate commit or not? I filled out a few, but I figured you might come up with better ones.

**Comment by MelonSmasher** [comment]
merged.

**Pull request merged**